go/vt/vtgate/engine/delete.go

```diff
@@ -57,9 +57,9 @@ func (del *Delete) TryExecute(ctx context.Context, vcursor VCursor, bindVars map
 
 	switch del.Opcode {
 	case Unsharded:
-		return del.execUnsharded(ctx, vcursor, bindVars, rss)
+		return del.execUnsharded(ctx, del, vcursor, bindVars, rss)
 	case Equal, IN, Scatter, ByDestination, SubShard, EqualUnique, MultiEqual:
-		return del.execMultiDestination(ctx, vcursor, bindVars, rss, del.deleteVindexEntries)
+		return del.execMultiDestination(ctx, del, vcursor, bindVars, rss, del.deleteVindexEntries)
```

Comment:
I think we do not need to pass primitive here, as the caller is the same primitive.